cdtb/tftdata.py

```diff
@@ -345,9 +346,13 @@ def main():
     parser = argparse.ArgumentParser()
     parser.add_argument("input", help="directory with extracted bin files")
     parser.add_argument("-o", "--output", default="tft", help="output directory")
+    parser.add_argument('-V', '--patch-version', default="14.15",
```

Comment:
We could use `99.99` or a special value (`latest` or `None`) for "last value".
This would be easier to maintain and use.

Reply:
I like having an actual patch version as the default value. It both signifies the format better than by using `99.99` (14.15 is a real patch version that exists, 99.99 isn't), and in this case carries meaning of 14.15 being a special value. The default value here is mainly for the user, not for the parser.

Reply:
It's weird to have 14.15 since we accept any other value. Typically you could expect to provide the actual current (or next for PBE) patch value. I agree that 99.99 is ugly. But we could just accept/use `None` or `"latest"` and convert it internally to 9999 (as in `PatchVersion`).

Reply:
> It's weird to have 14.15 since we accept any other value.

I don't understand what's weird about this. The same is done for the `bin-dump` command. Having the default value be in the same format as what we expect an input to be is good. Users can type `--help` and get an example for what value to provide, whereas having `None` as default gives no additional information. How should the `patch-version` argument be provided? `V14.5`? That won't parse. `14.5.1`? Also won't work. I feel like this default is a perfectly natural way to convey this information.

Reply:
The `patch-version` should be set to the patch version from which the file is from. The user shouldn't have to know that `14.15` is the last version with changes to the hash format, or when the hash format changed. cdtb will change handle that for them.

In any case, we should put an example value in the help message. But this does not have to be the default value (and we wouldn't have to update it if hash format changes).